etc/prepare-tests.sh

```diff
@@ -4,12 +4,6 @@ cd "$(dirname "$0")"/.. || exit 1
 
 outputDir="build/sync-functions"
 
-mkdir -p "$outputDir"
-mkdir -p build/test-reports/
```

Comment:
Moved to the `test-report` script in `package.json` since it is specific to that task.